iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/IidmVersion.java

```diff
@@ -83,25 +81,34 @@ public String getXsd(boolean valid) {
         return "iidm_equipment_V" + toString("_") + ".xsd";
     }
 
-    public static IidmVersion fromNamespaceURI(String namespaceURI) {
-        String version = namespaceURI.substring(namespaceURI.lastIndexOf('/') + 1);
-        IidmVersion v = of(version, "_");
-        String namespaceUriV = v.getNamespaceURI();
-        if (!namespaceURI.equals(namespaceUriV)) {
-            if (v.compareTo(V_1_7) >= 0 && namespaceURI.equals(v.getNamespaceURI(false))) {
-                return v;
-            }
-            throw new PowsyblException("Namespace " + namespaceURI + " is not supported. " +
-                    "The namespace for IIDM XML version " + v.toString(".") + " is: " + namespaceUriV + ".");
-        }
-        return v;
-    }
-
     public static IidmVersion of(String version, String separator) {
         Objects.requireNonNull(version);
         return Stream.of(IidmVersion.values())
                 .filter(v -> version.equals(v.toString(separator)))
                 .findFirst() // there can only be 0 or exactly 1 match
                 .orElseThrow(() -> new PowsyblException("Version " + version + " is not supported."));
     }
+
+    public static int compareVersions(String v1, String v2) {
```

Comment:
Usually, we use `v1.compareTo(v2)` to compare versions. This method may not be needed (and therefore neither `parseVersion(...)`).